src/modlist.cpp

```diff
@@ -1404,7 +1407,6 @@ bool ModList::toggleState(const QModelIndexList& indices)
 
   m_Profile->setModsEnabled(modsToEnable, modsToDisable);
 
-  emit modStatesChanged(indices);
```

Comment:
This isn't necessary because of dataChanged() or some other thing?

Reply:
I moved the signal to notifyModStateChanged which is called by the organizer after the changes to the directory structure, etc.

Reply:
I was searching how organizer knew about the change, but I'm updating VS hoping to make it stop crashing